app/views/hub/clients/show.html.erb

```diff
@@ -165,6 +165,14 @@
                     /
                     <span class="label-value"><%= "$#{benefits.claimed_recovery_rebate_credit}" if benefits.claimed_recovery_rebate_credit %></span>
                   </div>
+                  <% unless @client.intake.advance_ctc_amount_received.nil? %>
```

Comment:
is `benefits.outstanding_ctc_amount` guaranteed to be present if `@client.intake.advance_ctc_amount_received` is present?